packages/dnote/build.sh

```diff
@@ -1,37 +1,24 @@
 TERMUX_PKG_HOMEPAGE=https://www.getdnote.com/
 TERMUX_PKG_DESCRIPTION="A simple command line notebook for programmers"
 TERMUX_PKG_LICENSE="AGPL-V3,GPL-3.0"
-TERMUX_PKG_MAINTAINER="2096779623 <admin@utermux.dev>"
-TERMUX_PKG_VERSION=2.0.1
-TERMUX_PKG_REVISION=2
-TERMUX_PKG_SRCURL=https://github.com/dnote/dnote/archive/refs/tags/server-v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=28cc3bf93b3849b9a4a5e65e531f8a34d6be6048427d924c56ab8c7887676bad
+TERMUX_PKG_MAINTAINER="Ravener <ravener.anime@gmail.com>"
+TERMUX_PKG_VERSION=0.12.0
+TERMUX_PKG_SRCURL=https://github.com/dnote/dnote/archive/refs/tags/cli-v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=2b2c276f3b381a853e155ed2eab7007c5b5f2ec927b3d882d5605f2650b15085
 TERMUX_PKG_BUILD_IN_SRC=true
-TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_SUGGESTS="postgresql, dnote-server, dnote-client"
```

Comment:
Please add 

```
TERMUX_PKG_BREAKS=dnote-client
TERMUX_PKG_REPLACES=dnote-client
```

since files are moved from the `dnote-client` package to `dnote`.

(See https://www.debian.org/doc/debian-policy/ch-relationships.html#conflicting-binary-packages-conflicts)